tests/actions/action_engine_test.py

```diff
@@ -0,0 +1,117 @@
+# Copyright 2017 The Forseti Security Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import mock
+import os
+import unittest
+
+from google.cloud.forseti.services.actions import action_engine as ae
+from tests.unittest_utils import ForsetiTestCase
+from tests.actions import action_config_data
+
+
+class ActionTest(ForsetiTestCase):
+
+  def setUp(self):
+    self.action = ae.Action({}, 'id', 'type')
+
+  def test_from_dict(self):
+    with self.assertRaises(NotImplementedError):
+      ae.Action.from_dict({})
+
+  def test_act(self):
+    with self.assertRaises(NotImplementedError):
+      self.action.act('result')
+
+
+class SampleActionTest(ForsetiTestCase):
+
+  def setUp(self):
+    self.success_action = ae.SampleAction({}, 'id', 'type', 0, 'info1')
```

Comment:
These are only used in test_act, I would create them in test_act so the setup is near the test.